helm/azure-resourcemanager-exporter/templates/_helpers.tpl

```diff
@@ -0,0 +1,53 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "azure-resourcemanager-exporter.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+App name is truncated to 63 chars because some Kubernetes name fields are limited to this (by DNS naming spec).
```

Comment:
```suggestion
We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
```

Reply:
This is the default Helm message from helm create. We can keep it as it defaults.